### PR TITLE
UI Changes  on Home page and admin sidebar

### PR DESCRIPTION
**Pull request by vaibhav8186**
removed sidebar at admin side and managed on navbar ,
terminal pages some ui chanes like - progress bar, confirmation possibility text font etc.

### CODE REVIEW

File: app/assets/stylesheets/orders.scss [hunk 1]
```diff
@@ -98,3 +98,8 @@ input[type=number]::-webkit-outer-spin-button {
   position: sticky;
   top: 0;
 }
+
+#order_btn{
```
Comment (kirandeshmukh) [inline app/assets/stylesheets/orders.scss]: Keep ids and class-names consistent. We are already using  "dash" between them (e.g. ht-middle-section). Please follow them

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -43,6 +43,7 @@ h1, h2, h3, h4, h5, h6 {
 p {
   font-size: 1.1em;
   line-height: 1.7em;
+  margin-left:1%;
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/assets/stylesheets/application.scss]: done this

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -43,6 +43,7 @@ h1, h2, h3, h4, h5, h6 {
 p {
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/assets/stylesheets/application.scss]: ok. i will change this

File: app/assets/stylesheets/orders.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/assets/stylesheets/orders.scss]: ok done this

File: app/assets/stylesheets/orders.scss [hunk 4]
```diff
@@ -98,3 +98,8 @@ input[type=number]::-webkit-outer-spin-button {
...
+  padding: 0 45px;
+  border-radius: 22px;
+}
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/assets/stylesheets/orders.scss]: done

File: app/views/orders/load_terminal.html.haml [hunk 5]
```diff
@@ -6,44 +6,60 @@
...
-                = link_to "Order", new_terminal_order_path(terminal.id), class: "btn grey darken-4"
+              .bottomFixed
+                = link_to "Order", new_terminal_order_path(terminal.id), class: "btn order-btn"
 
 %script{:type => 'text/javascript'}
   orderTime = "#{@end_ordering_at}"
   today = new Date()
   today = `${today.getFullYear()}/${today.getMonth()+1}/${today.getDate()}`
-  $('#order-timer').countdown(`${today} ${orderTime}`, function(event) {
+  $('#order-timer').countdown(`${today} ${orderTime}`, function(event) {
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/views/orders/load_terminal.html.haml]: ok

File: app/views/orders/load_terminal.html.haml [hunk 6]
```diff
@@ -6,44 +6,60 @@
...
+  $('#order-timer').countdown(`${today} ${orderTime}`, function(event) { 
+  
   $(this).html(event.strftime('%H:%M:%S'));
+  if(event.strftime('%M') > "30"){
+  document.getElementById('order-timer').className = " green-text "
+  }
+  else{
+  document.getElementById('order-timer').className = " red-text " 
+  }
   });
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/views/orders/load_terminal.html.haml]: ok

File: app/views/users/show.html.haml [hunk 7]
```diff
@@ -5,7 +5,7 @@
   %h3.center-align Employee Record
   %br
   %br
-  %table.table-hover{:border =>3,:align=>"center",:style=>"width:60%;background-color:white"}
+  %table.table-hover{:border =>3,:align=>"center",:style=>"width:60%;background-color:white;margin-left:25%"}
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/views/users/show.html.haml]: ok

File: app/assets/stylesheets/orders.scss (hunk 1 above)
Comment (vaibhav8186) [reply app/assets/stylesheets/orders.scss]: ok

File: app/views/home/_navbar.html.haml [hunk 8]
```diff
@@ -8,11 +8,52 @@
       %i.material-icons menu
     %ul.right.hide-on-med-and-down
       %li
-        -if user_signed_in? && current_user.role == "company_admin" 
-          %a{:href => get_order_details_company_path(current_user.company_id)}
-            Set Order details 
-          %span.sr-only (current)
-        -if user_signed_in?    
+        -if user_signed_in? && current_user.role == "company_admin"
```
[the remark this answers is not in the document]
Comment (vaibhav8186) [reply app/views/home/_navbar.html.haml]: yes we can but already there is one function call and we just added one condition

**Pull request closed without merging**